src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        int ball1 = lottery.getRandomBall();
+        int ball2 = lottery.getRandomBall();
+        int ball3 = lottery.getRandomBall();
+
+        System.out.println(ball1);
+        System.out.println(ball2);
+        System.out.println(ball3);
```

Comment:
Repeatedly calling `System.out.println` is not efficient. Consider using a loop to iterate over the collection of balls and print them in a single loop. This will make your code cleaner and reduce duplication.